src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,78 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { CommentData } from '../types/Comment';
+
+type Props = {
+  buttonLoader: boolean;
+  addFunction: (newComment: CommentData) => Promise<void>;
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  buttonLoader,
+  addFunction = () => {},
```

Comment:
Setting a default value for `addFunction` to an empty function can lead to unexpected behavior if the prop is not provided. Consider making this prop required without a default value.